### PR TITLE
Fix deprecated class query

### LINKED ISSUE #884: ROBOT report: broken deprecated_class_reference check
I just noticed something odd :

When I download the deprecated_class_reference check from http://robot.obolibrary.org/report_queries/deprecated_class_reference I get a lot of (correct) results:

```
robot --version
ROBOT version 1.8.1
robot query -i hp-edit.owl --query deprecated_class_reference.sparql dep.txt
```

When I run robot report:
```
robot report -i hp-edit.owl -o report.txt
```

I get none. On closer inspection, the docs were simply not updated, but the check (sparql query) was.. And it now has a few issues:

This seems to make no sense:

```
   FILTER EXISTS { ?property a owl:ObjectProperty }
   FILTER EXISTS { ?property a owl:DatatypeProperty }
```

There are many other problems with this query as well, I think its high priority to fix..

### PR DESCRIPTION
**Pull request by beckyjackson**
Resolves #884

- [x] `docs/` have been added/updated
- [ ] tests have been added/updated
- [ ] `mvn verify` says all tests pass
- [ ] `mvn site` says all JavaDocs correct
- [ ] `CHANGELOG.md` has been updated

The query was actually missing the part that checks if the deprecated class is a subclass of something else, it was only checking if *something else* was a subclass of the deprecated class. I also changed the `FILTER EXISTS` to use `VALUES` instead to match the other unions.

I ran `make all` in `docs/` and it looks like the queries in the docs are quite out of date. We should make sure to run this every time we update a report query.

### CODE REVIEW

**Comment by beckyjackson** [comment]
@matentzn when you get a chance, can you see if this works for you?

**Comment by matentzn** [comment]
The query does not terminate on large ontologies.. Looking into it, but maybe you can spot something

**Comment by beckyjackson** [comment]
@jamesaoverton this PR is ready for review, but it binds the blank node as "blank node" (which we agreed to switch to). Do you want me to change all the blank nodes to "blank node" in this same PR? Or should that be done separately?

**Comment by matentzn** [comment]
I would say lets do this all at once - I can also do it, your time is maybe a bit too precious for such menial tasks :D Up to you. But since we define the new default behavior here (binding to `blank node` if blank) we should probably apply this consistently.

**Comment by jamesaoverton** [comment]
Ok. @matentzn please update all the queries to use "blank node". Thanks!

**Comment by matentzn** [comment]
@beckyjackson before I get into this: the branch here is so massively add odds with master - does it have to be redone? I am not sure exactly why what has changed where - let me know if you prefer me to redo, or whether you can fix conflicts?

**Comment by beckyjackson** [comment]
I tried to remake the docs, but it looks like something has gone wrong with the new changes to the Makefile. Let's abandon this branch and make the changes to the deprecated class query in a new branch. Do you want to do that or should I @matentzn?

File: docs/Makefile [hunk 1]
```diff
@@ -31,7 +31,8 @@ report_queries:
 # Convert each SPARQL file to a Markdown file.
 report_queries/%.md: ../robot-core/src/main/resources/report_queries/%.rq | report_queries
 	awk '/^PREFIX/ && !x {print "```sparql"; x=1} 1' $< \
-	| sed 's/^# \? \?//' \
+	| sed 's/^#//' \
+	| sed 's/^ //' \
```
Comment (jamesaoverton) [inline docs/Makefile:35]: FYI I changed this bit of the `docs/Makefile` on another branch to make the indentation better, which changes a all the `docs/report_queries/*.md` files.

**Comment by matentzn** [comment]
My worry is that this branch has a bit more than just the changes to the deprecated query.. Look for example at this commit (and some [more](https://github.com/ontodev/robot/pull/888/commits)):
https://github.com/ontodev/robot/pull/888/commits/70d1a42832cad96f7fbe9c621d40182ad84913bf

I think I would lose orientation if I had to replicate the branch myself.. If you could do it, and believe it wouldn't take you too long, I would feel safer if you did it...

**Comment by beckyjackson** [comment]
Closed in favor of #902

**Pull request closed without merging**